source/common/http/filter_manager.cc

```diff
@@ -1000,10 +1000,12 @@ void DownstreamFilterManager::sendLocalReply(
       // route refreshment in the response filter chain.
       cb->route(nullptr);
     }
-
+  
+    bool reverse_conn_force_local_reply = 
```

Comment:
We perfer latching the runtime value to avoid the cost to get it every time sending a local reply.